src/lib/flows/claim-project-flow/steps/enter-git-url/enter-git-url.svelte

```diff
@@ -208,20 +202,17 @@
     {/if}
   {/if}
   <svelte:fragment slot="actions">
-    <Button
-      disabled={!formValid}
-      icon={ArrowRightIcon}
-      variant="primary"
-      on:click={() => dispatch('goForward')}>Continue</Button
-    >
+    {#if formValid}
+      <Button icon={ArrowRightIcon} variant="primary" on:click={() => dispatch('goForward')}
+        >Continue</Button
```

Comment:
```suggestion
        >Claim</Button
```

Reply:
Hi, thanks for your enthusiasm. If you'd like to contribute to Drips, tackling some of our ['good first issues'](https://github.com/drips-network/app/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) would be most helpful. Also, we don't merit approvals from outside contributors so please focus your energy elsewhere.